android/java/AndroidManifest_activity_alias.xml

```diff
@@ -0,0 +1,179 @@
+<activity-alias
+        android:name=".Icon3dAlias"
+        android:enabled="false"
+        android:icon="@mipmap/ic_launcher_3d"
+        android:roundIcon="@mipmap/ic_launcher_3d_round"
+        android:label="@string/app_name"
+        android:targetActivity="org.chromium.chrome.browser.ChromeTabbedActivity"
+        android:exported="true">
+        <intent-filter>
+            <action android:name="android.intent.action.MAIN" />
+            <category android:name="android.intent.category.LAUNCHER" />
+            <category android:name="android.intent.category.APP_BROWSER" />
```

Comment:
I wonder shouldn't we put other intents that are in `android:name="com.google.android.apps.chrome.Main"`
```
                <category android:name="android.intent.category.APP_BROWSER" />
                <category android:name="android.intent.category.MULTIWINDOW_LAUNCHER" />
```

Reply:
@SergeyZhukovsky i verified the intent but i think it's safer to include everything we have in `com.google.android.apps.chrome.Main` not just intent filters. updating with a commit now.

Reply:
Check please also `meta-data` especially `xml/launchershortcuts` that provides Open in tabs by tapping and holding on the icons. Do they work?

Reply:
yeah i am everything from `com.google.android.apps.chrome.Main`

Reply:
that
```
{{ self.supports_vr() }}
{{ self.extra_web_rendering_activity_definitions() }}
```
is also probably needed, not sure could we do such template blocks in our xml file, probably yes.

Reply:
that's exactly what i am checking right now.

Reply:
i analyzed the apk manifest, and based on that, we need to add below for,
{{ self.supports_vr() }}
{{ self.extra_web_rendering_activity_definitions() }}

`<intent-filter>
                <action
                    name="org.chromium.chrome.browser.dummy.action" />
                <category
                    name="com.google.intent.category.DAYDREAM" />
                <category
                    name="com.google.intent.category.CARDBOARD" />
            </intent-filter>`